app/controllers/states_controller.rb

```diff
@@ -2,5 +2,6 @@ class StatesController < ApplicationController
   def index
     state_info = State.order(:name).pluck(:name, :id)
     @names_by_letter = state_info.group_by { |state| state[0][0] }
+    @occupation_standards_count_by_state = OccupationStandard.joins(registration_agency: :state).group("states.id").count
```

Comment:
@jenlu33 let's make this count a separate method in the model, and then you can add some unit tests for it.

Reply:
Maybe we could also beef up the index request spec making sure there are at least 2 of the same states to count so Bullet can help catch any query inefficiencies.